R/cluster.R

```diff
@@ -156,13 +157,15 @@ makeCluster <-
 
     # log into private registry if registry credentials were provided
     if (!is.null(config$dockerAuthentication) &&
-        !is.null(config$dockerAuthentication$username)) {
+        !is.null(config$dockerAuthentication$username) &&
+        nchar(config$dockerAuthentication$username) > 0 &&
+        nchar(config$dockerAuthentication$password) > 0 &&
```

Comment:
You can remove the !is.null(config$dockerAuthentication$username) since nchar(NULL) > 0 will cover that case for you

Reply:
Done.